smif/convert/area.py

```diff
@@ -40,6 +39,9 @@ class RegionSet(ResolutionSet):
     """
     def __init__(self, set_name, fiona_shape_iter):
         self._name = set_name
+        self._description = ''
```

Comment:
For conciseness, these could just be public attributes - with the option to move to property/setter methods if and when we need to do extra work while setting/getting.